tools/justfile

```
@@ -0,0 +1,144 @@
+# Enable positional args
+set positional-arguments
+
+# Default values for the deployment
+tag    := "test"
+queue  := "active"
+worker := "htcmock"
+
+# Base compose file
+export COMPOSE_BASE := "../docker-compose/docker-compose.yml"
+
+# loggin level of deployment
+export LOGGING_LEVEL := "Information"
+
+# Sets the queue
+export QUEUE := if queue == "rabbit" {
+  "../docker-compose/docker-compose.queue-rabbitmq.yml"
+}else if queue == "rabbit091"{
+  "../docker-compose/docker-compose.queue-rabbitmq.yml"
+}else if queue == "artemis"{
+  "../docker-compose/docker-compose.queue-artemis.yml"
+}else{
+  "../docker-compose/docker-compose.queue-activemqp.yml"
+}
+
+
+# Sets the override to feed docker-compose
+export OVERRIDE := if queue == "rabbit091" {
+  "../docker-compose/docker-compose.override-rabbitmq091.yml"
+}else{
+  "../docker-compose/docker-compose.override.yml"
+}
+
+# Defines worker and enviroment variables for deployment
+export ARMONIK_WORKER := if worker == "stream" {
+  "dockerhubaneo/armonik_core_stream_test_worker:"+tag
```

I think you can add whitespaces around `+`
```suggestion
  "dockerhubaneo/armonik_core_stream_test_worker:" + tag
```